unattended-upgrade

```diff
@@ -817,6 +844,15 @@ def check_changes_for_sanity(cache, allowed_origins, blacklist, whitelist,
                 return False
             if not is_pkg_change_allowed(pkg, blacklist, whitelist):
                 return False
+            if (pkg.marked_install
+                and "dkms" in transitive_dependencies(pkg, cache)
```

Comment:
@vorlonofportland this is a bit different workaround than suggested in https://bugs.launchpad.net/ubuntu/+source/shim-signed/+bug/1726803